publishexe.targets

```diff
@@ -38,6 +37,11 @@
   <Target Name="FixUpCoreConsoleNames">
     <Move SourceFiles="$(Outdir)$(AssemblyName).exe" DestinationFiles="$(Outdir)$(AssemblyName).dll" />
     <Move SourceFiles="$(Outdir)CoreConsole.exe" DestinationFiles="$(Outdir)$(AssemblyName).exe" />
+    <!-- create a marker indicating that the file was renamed, this will be used during signing to sign the correct file -->
+    <Touch Files="$(Outdir)$(AssemblyName).publishedexe" AlwaysCreate="true" />
+    <ItemGroup>
```

Comment:
I've removed this marker and updated the TargetPath property (I swear I tried this earlier and it didn't work, glad it's working now as it's much simpler).